Dockerfile.ci

```diff
@@ -1,16 +1,21 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19 as builder
 WORKDIR /go/src/github.com/openshift/lws-operator
 COPY . .
+RUN make build --warn-undefined-variables
 
-RUN make build
-
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-base-openshift-4.19
+FROM registry.ci.openshift.org/ocp/builder:base-rhel9
 COPY --from=builder /go/src/github.com/openshift/lws-operator/lws-operator /usr/bin/
-COPY --from=builder /go/src/github.com/openshift/lws-operator/manifests/* /manifests/
+RUN mkdir /licenses
+COPY --from=builder /go/src/github.com/openshift/lws-operator/LICENSE /licenses/.
 
-LABEL io.k8s.display-name="Leader Worker Set" \
-      io.k8s.description="This is a component of OpenShift and manages the leader worker set" \
+LABEL com.redhat.component="Leader Worker Set"
```

Comment:
since this is only for the CI, the labels do not matter much, right?

Reply:
Yeah, just for consistency